subscription_integration_test.go

```diff
@@ -47,6 +47,25 @@ func TestSubscriptionSimple(t *testing.T) {
 	if sub.Id == "" {
 		t.Fatal("invalid subscription id")
 	}
+	if len(sub.StatusEvents) != 1 {
+		t.Fatalf("expected one status event, got %d", len(sub.StatusEvents))
+	}
+	wantBalance := NewDecimal(0, 2)
+	wantPrice := NewDecimal(1000, 2)
+	for _, event := range sub.StatusEvents {
```

Comment:
Because there's only one status event and you check that a few lines above, we can drop the for loop here and do `sub.StatusEvents[0]`.